client/my-sites/people/delete-user/index.jsx

```diff
@@ -125,7 +125,7 @@ class DeleteUser extends React.Component {
 	getAuthorSelectPlaceholder = () => {
 		return (
 			<span className="delete-user__select-placeholder">
-				<User size={ 26 } user={ { name: /* Don't translate yet */ 'Choose an author…' } } />
+				<User size={ 26 } user={ { name: this.props.translate( 'Choose an author…' ) } } />
```

Comment:
I think this is fine! I'm not sure why we wouldn't have wanted to translate the value here. It might be worth pinging for an i18n review.

Reply:
It looks like this code is from 2 years ago... maybe this was just a new feature and we weren't planning to roll it out and this was missed? Just guessing there though.